Makefile

```diff
@@ -9,14 +9,19 @@ elixir_%:
 	$(MAKE) -C implementations/elixir $(@:elixir_%=%)
 
 rust_%:
-	$(MAKE) -C implementations/rust $(@:rust_%=%)
+	$(MAKE) -f implementations/rust/Makefile $(@:rust_%=%)
+
+# run a rust command in a nix environment, so that all tools are installed
+nix_rust_%:
+	nix develop ./tools/nix#rust --command make rust_$*
 
 swift_%:
-	$(MAKE) -C implementations/swift $(@:swift_%=%)
+	$(MAKE) -f implementations/swift/Makefile $(@:swift_%=%)
```

Comment:
Let's revert this change as it breaks release, using the -C flag ensures we cd to the swift directory before running the make commands, which is crucial for building the desktop app
https://github.com/build-trust/ockam/blob/4862e4ae1a9cb9dcf0a2604482be2b05cb676c63/implementations/swift/Makefile#L2

Reply:
Done